<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinTestPlot</class>
 <widget class="QDialog" name="WinTestPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>410</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test Results</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_n_reads">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>N°Reads:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_b_writes">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>N°Writes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rw_type">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>150</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RW Type:</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_type_test">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Average values</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_compare">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
